zendesk/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "zendesk",
+  "title": "Zendesk — Search Support Center",
+  "description": "Search Zendesk support center for articles.",
+  "icon": "zendesk.png",
+  "author": "cpenned",
+  "license": "MIT",
+  "categories": [
+    "Productivity",
+    "Data"
+  ],
+  "commands": [
+    {
+      "name": "search-zendesk-support-center",
+      "title": "Search Support Center",
+      "subtitle": "Zendesk",
+      "description": "Search any Zendesk support center.",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "supportCenter",
+      "type": "textfield",
+      "required": true,
+      "title": "Support center base URL",
+      "placeholder": "e.g., support.mycompany.com",
+      "description": "The base URL for your Zendesk support center. When viewing your support articles online, the URL will be in the format https://support.mycompany.com/hc/en-us/articles/3600000000000-My-Article-Title. The base URL is everything after the https:// and before the /hc/ part of the URL."
+    },
+    {
+      "name": "locale",
+      "type": "textfield",
+      "required": false,
+      "title": "Support center locale",
+      "placeholder": "e.g., en-us",
+      "description": "Defaults to en-us if no locale is provided."
```

Comment:
Are all locales supported? Thinking whether we could display this in a [dropdown](https://developers.raycast.com/information/manifest#preference-properties) instead of textfield

Reply:
Ah, that's a good point. Looking closer into the Zendesk documentation, there's a way to fetch which languages are available for each support center. Perhaps I can make it a searchBarAccessory dropdown? And then whatever they select, in the dropdown persists over uses? Now I just need to figure out how to do that if that's the right direction?

Reply:
Yeah, I think the `Dropdown` Component could work well here 👌